app/lib/screens/EditorScreen/Editor.dart

```diff
@@ -0,0 +1,320 @@
+import 'package:effektio/common/store/Colors.dart';
+import 'package:effektio/common/widget/TagItem.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:html_editor_enhanced/html_editor.dart';
+
+void main() => runApp(const HtmlEditorExampleApp());
+
+var arr = [
+  Colors.grey,
+  Colors.purple[300],
+  Colors.pink[100],
+  Colors.teal[400],
+  Colors.purple[700],
+  Colors.orange,
+  Colors.blue[300],
+  Colors.green,
+  Colors.yellow[100],
+  Colors.yellow,
+  Colors.pink,
+  Colors.red
+];
+
+List<String> _tagList = [];
+
+class HtmlEditorExampleApp extends StatelessWidget {
+  const HtmlEditorExampleApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Editor',
+      theme: ThemeData.dark(),
+      darkTheme: ThemeData.dark(),
+      home: const HtmlEditorExample(title: 'Create FAQ'),
+    );
+  }
+}
+
+class HtmlEditorExample extends StatefulWidget {
+  const HtmlEditorExample({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _HtmlEditorExampleState createState() => _HtmlEditorExampleState();
+}
+
+class _HtmlEditorExampleState extends State<HtmlEditorExample> {
+  String result = '';
+  OverlayEntry? entry;
+  TextEditingController tagTitleController = TextEditingController();
+
+  final HtmlEditorController controller = HtmlEditorController();
+
+  @override
+  void initState() {
+    super.initState();
+    _tagList.clear();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        if (!kIsWeb) {
+          controller.clearFocus();
+        }
+      },
+      child: Scaffold(
```

Comment:
Tested on Android 9.0 device, I'm not sure if you've encountered this but opening an editor crashes the application and and closes immediately.

```
Abort message: 'Invalid address 0x7e89d0d0c0 passed to free: value not allocated'
    x0  0000000000000000  x1  0000000000005518  x2  0000000000000006  x3  0000000000000008
    x4  8000000000808080  x5  8000000000808080  x6  8000000000808080  x7  0000000000000008
    x8  0000000000000083  x9  4969343f664fd22c  x10 0000000000000000  x11 fffffffc7ffffbdf
    x12 0000000000000001  x13 00000000627ec97c  x14 000aa4296bdb1a28  x15 00000e09f4154c22
    x16 0000007f3bbb12c0  x17 0000007f3baefe34  x18 0000000000000001  x19 0000000000005286
    x20 0000000000005518  x21 0000007e898fec88  x22 0000000000000000  x23 0000007f3bbbdc20
    x24 0000007f3bbbdc9c  x25 0000000000000000  x26 0000007e7f548f00  x27 0000000000000000
    x28 0000007e970b0860  x29 0000007e971f71d0
    sp  0000007e971f7190  lr  0000007f3bae4960  pc  0000007f3bae4988
backtrace:
    #00 pc 0000000000022988  /system/lib64/libc.so (abort+116)
    #01 pc 00000000000a0db0  /system/lib64/libc.so (ifree+1204)
    #02 pc 00000000000a0edc  /system/lib64/libc.so (je_free+120)
    #03 pc 00000000019974d8  /data/app/org.effektio.app-rf_Gew1qyCwant2OtQlgjA==/lib/arm64/libflutter.so (offset 0x1507000)
    #04 pc 0000000000000001  <unknown>
Lost connection to device.
Exited (sigterm)
```